### PR TITLE
Update PuzzleStealth.md

### PR DESCRIPTION
**Pull request by and-thomp**
Adding goals for my management week

### CODE REVIEW

File: management/PuzzleStealth.md [hunk 1]
```diff
@@ -15,9 +15,10 @@
 3. 9/16 - 9/22
 	* Manager: Andrew Thompson
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		1. Player movement with a static camera
+		2. Player sprite
+		3. walls and stuff
```
Comment (nfarnan) [inline management/PuzzleStealth.md:20]: Need more details here, are you just displaying walls? Implementing collision detection against all walls?

File: management/PuzzleStealth.md [hunk 2]
```diff
@@ -15,9 +15,10 @@
 3. 9/16 - 9/22
 	* Manager: Andrew Thompson
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		1. Player movement with a static camera
+		2. Player sprite
```
Comment (nfarnan) [inline management/PuzzleStealth.md:19]: Doing what with the player sprite?

File: management/PuzzleStealth.md [hunk 3]
```diff
@@ -15,9 +15,10 @@
 3. 9/16 - 9/22
 	* Manager: Andrew Thompson
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		1. Player movement with a static camera
```
Comment (nfarnan) [inline management/PuzzleStealth.md:18]: How much movement will be implemented?

File: management/PuzzleStealth.md [hunk 4]
```diff
@@ -15,9 +15,10 @@
...
+		2. Player sprite
+		3. walls and stuff
+		4. Game loop
```
Comment (nfarnan) [inline management/PuzzleStealth.md:21]: Are you going to write the entire final contents of the game loop? If not, what, specifically do you want to have for the game loop this week?

**Review by nfarnan — CHANGES_REQUESTED** [review]
Overall need more specifics in your goal specification

**Review by and-thomp — COMMENTED** [review]
I added detail to the goals for the week

File: management/PuzzleStealth.md [hunk 5]
```diff
@@ -15,9 +15,10 @@
 3. 9/16 - 9/22
 	* Manager: Andrew Thompson
 	* Goals:
-		1. GOAL1
-		1. GOAL2
-		1. GOAL3
+		1. Player movement with a static camera - player will be in the center of the screen with the map moving around them
+		2. Player sprite - design the sprite to display in the middle of the screen during gameplay
+		3. walls and stuff - basic collision detection and laying out a basic map to test with
```
Comment (nfarnan) [inline management/PuzzleStealth.md:20]: With everything else, let's drop collision detection as a requirement. If you get to that as well, great, but supporting a main menu and movement with a static camera and moving map will be enough for the first real game code management week.

**Review by nfarnan — CHANGES_REQUESTED** [review]

**Comment by and-thomp** [comment]
I removed that goal, now we have the player movement, the sprite, and the menu

**Review by nfarnan — APPROVED** [review]

**Pull request merged**